src/main/scala/rx/lang/scala/JavaConverters.scala

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2014 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package rx.lang.scala
+
+import scala.language.implicitConversions
+import Decorators.AsJava
+import Decorators.AsScala
+import Decorators.AsJavaSubscription
+
+/**
+	* Provides conversion functions `asJava` and `asScala` to convert
+	* between RxScala types and RxJava types.
+	*
+	* Example:
+	* {{{
+	* import rx.lang.scala.JavaConverters._
+	* val javaObs = Observable.just(1, 2, 3).asJava
+	* val scalaObs = javaObs.asScala
+	* }}}
+	*/
+object JavaConverters extends DecorateAsJava with DecorateAsScala
+
+private[scala] trait Decorators {
```

Comment:
(sorry for nitpicking) Code indentation in this project is 2 spaces, could you please adhere to this?